### PR TITLE
Individual expand topics

### PR DESCRIPTION
**Pull request by yougotdirked**
- added individual expand buttons per topic block
- moved expand/collapse button to head of blocks list

We probably need to style the button to its new place (bigger? text explanation?) but I'm hesitant to put it in this PR. I can add it though if it does fit.

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html lang="{{lang}}">
   <head>
+    <!-- content="
+    default-src 'none';
+    font-src 'none';
+    img-src 'self';
+    script-src 'self' 'sha256-QIO/JKO6nn9l5q1faq4A06sEzbRLkirNwdq3+qJzJVQ=';
+    style-src 'self';
+  " -->
```
Comment (drikusroor) [inline template.html:10]: You forgot to uncomment this 😉 You can generate a new hash here: https://csplite.com/csp/sha/

File: template.html [hunk 2]
```diff
@@ -92,6 +73,28 @@ <h2><a href="#infi-test">
...
+                <div></div>
+                <div></div>
+              </div>
+              <div class="list-view">
+                <div></div>
+                <div></div>
+              </div>
+              <span class="sr-only">{{utilityBar.toggleView}}</span>
+            </button>
+        </nav>
```
Comment (drikusroor) [inline template.html:97]: 1. As you said, this button could have an explanatory text accompanying the icon. The space is also there for it.
2. You might want to use a `+` and `-` or chevron down / up icon for it now.

**Review by drikusroor — CHANGES_REQUESTED** [review]
Nice!
I would at least generate a new hash for the CSP part. Other than that you could add a text and change the icon for the toggle all button. For the rest it LGTM 👍

File: template.html (hunk 2 above)
Comment (yougotdirked) [reply template.html:97]: I figured we can improve button styling in another PR. Maybe when we do a styling pass to unify the design more.

**Review by drikusroor — APPROVED** [review]

**Pull request merged**